app/src/main/java/com/asfoundation/wallet/wallet/home/HomeViewModel.kt

```diff
@@ -229,7 +256,7 @@ class HomeViewModel @Inject constructor(
     if (walletModel == null) return Observable.empty()
     val retainValue = if (walletModel.isNewWallet) null else HomeState::transactionsModelAsync
     return Observable.combineLatest(
-      getTransactions(walletModel.wallet), getCardNotifications(),
+      Observable.just(listOf()), getCardNotifications(),
```

Comment:
What is the purpose of this change?

Reply:
It was just to stop fetching the transactions from the another api? I can remove it totally, this approach was just to avoid bugs in other flows

Reply:
It looks weird to have a Observable to do nothing, it created unecessary memory allocation (assuming that the previous code is not needed to get the transactions anymore).
If its not need, I would say to simply remove it and since this is inside the HomeViewModel, im assuming no other flow will break because of it.
Especially because passing this empty list as transactions to other methods might generate issues in debug because it looks like that might return the transactions and in this case the TransactionsModel is the only think affected and it appears to be used in the old flow home, but in this branch the old home should be removed anyways, so we can even remove this TransactionsModel if its not being used

Reply:
I got it, already done!